pkg/operator/ceph/pool/controller.go

```diff
@@ -429,10 +429,10 @@ func (r *ReconcileCephBlockPool) handleDeletionBlocked(cephBlockPool *cephv1.Cep
 	logger.Info(depCondition.Message)
 
 	radosNamespaces := deps.OfKind(radosNamespacesKeyName)
-	isEmpty, emptyMessage, err := cephclient.IsPoolEmpty(r.context, r.clusterInfo, poolSpec.Name, radosNamespaces)
-	if err != nil {
-		return err
-	}
+	isEmpty, emptyMessage, deleteErr := cephclient.IsPoolEmpty(r.context, r.clusterInfo, poolSpec.Name, radosNamespaces)
+	// If err is not nil, it means the deletion failed, but we still want to
```

Comment:
If err is not nil, I believe it means we hit this error:
```
failed to list images/snapshots in pool %s
```
So I don't think we know if there are images. But agreed it would be preferred to update the condition to indicate we don't know.